### ISSUE TITLE
Ensure all links on the website don't include the full stop at the end of the sentence

### ISSUE THREAD

**Issue by GCHQ-Developer-847** [mi6/ic-design-system#1057]
## Summary
Some links on the website which are at the end of a sentence include the full stop within the link. The full stop needs to be moved to after the link instead.

## 💬 Description
See these pages, for example:
- [Home page](https://design.sis.gov.uk)
- [Link page in the Styles section](https://design.sis.gov.uk/styles/content-style/links)

This goes against the [guidance provided on the website about using links](https://design.sis.gov.uk/styles/typography#links) and also appears to be [an accessibility issue](https://jimbyrne.co.uk/link-falls-end-sentence-put-full-stop-outside-anchor-tag/) (particularly when the next sentence begins with a link).

## 💰 Use value
This should be fixed as it may cause confusion to screen reader users with how adjacent links are read out. Also the content of the website is not following the actual guidance that it provides.

## Additional info
After a quick look, there doesn't seem to be too many but it may require looking through all the pages to double-check all the links.
